src/aws/dynamodb-utils.ts

```diff
@@ -1,30 +1,28 @@
 import AWS from 'aws-sdk';
 import {
   BatchWriteItemInput,
-  BatchWriteItemOutput,
   BatchWriteItemRequestMap,
   DocumentClient,
   PutItemInput,
-  PutItemOutput,
   QueryInput,
 } from 'aws-sdk/clients/dynamodb';
 import AttributeValue = DocumentClient.AttributeValue;
 
 const ddb = new AWS.DynamoDB.DocumentClient();
 
-export const saveItem = async (table: string, item: AttributeValue): Promise<PutItemOutput> => {
+export const saveItem = async (table: string, item: AttributeValue): Promise<void> => {
   const params: PutItemInput = {
     TableName: table,
     Item: item,
   };
-  return ddb.put(params).promise();
+  await ddb.put(params).promise();
```

Comment:
this response provided no created items data as i thought, pointless to return anything here

Reply:
you'd have to await it further out...

Reply:
this one is a void type return for now, will add params for return types on this one and revert to `return ...` once its no longer void